tutorials/s3cmd/index.mdx

```diff
@@ -41,13 +41,11 @@ categories:
   - storage
   - object-storage
 dates:
-  validation: 2021-05-10
+  validation: 2021-12-01
   posted: 2018-06-04
 ---
```

Comment:
I don't know if we can rename the slug into something more useful like `how-to-use-object-storage-with-s3cmd` ?